modules/drivers/bigquery-cloud-sdk/src/metabase/driver/bigquery_cloud_sdk/query_processor.clj

```diff
@@ -182,10 +198,10 @@
   :time)
 
 (defmethod temporal-type :field
-  [[_ id-or-name {:keys [base-type temporal-unit], :as opts} :as clause]]
+  [[_ id-or-name {:keys [base-type temporal-unit]} :as clause]]
   (cond
-    (contains? (meta clause) :bigquery-cloud-sdk/temporal-type)
-    (:bigquery-cloud-sdk/temporal-type (meta clause))
+    (contains? (meta clause) :bigquery-cloud-sqk/temporal-type)
```

Comment:
looks like a typo here: `bigquery-cloud-sQk/temporal-type`

Reply:
there are four instances of this keyword namespace in this code namespace and nowhere else. Did some automatic renaming go awry here?

Reply:
Yes, oops. I don't know how everything still worked. Uh oh

Reply:
I'm guessing, at least for this method, it's because the same typo was assigned as checked?  So the namespaced keyword `:bigquery-cloud-sqk/temporal-type` was used for both, and therefore worked as expected?

Reply:
Fixed